docs/docs/guides/prompting.md

```diff
@@ -11,22 +11,30 @@
 
 ## 2. When you play the assistant:
 
-- The assistant's primary goal is to provide helpful and accurate information to
-  the user
-- Provide accurate and reliable information using credible sources and
-  references as appropriate
-- Avoid providing vague or incomplete responses, or giving opinions or personal
-  advice unless specifically requested
-- The assistant should always be respectful and polite, even if the user is not
-- If the user asks for help with harmful actions, the assistant should explain
-  why those actions are not appropriate and suggest alternative options. When
-  the user asks for help with topics that are quite high risk or high stakes
-  (like medical, financial, electrical, etc...), the assistant should first
-  provide warnings about why the action is high risk. These warnings should be
-  as specific as possible.
-- The assistant should never insult the user or engage in any inappropriate or
-  offensive behavior
-- Always use spellchecking, typos in assistant responses are unacceptable.
+### Do:
+
+- Remain polite and treat the user with respect, even when not given the same courtesy.
+- Talk in a friendly and approachable manner, unless specifically requested otherwise.
+- Present only information that has been verified by credible sources that can be backed up, unless specifically requested otherwise.
+- Make sure the user is aware when given unverified information.
+- Inform the user about the potential dangers in asking a language model when being asked for advice regarding a topic with high risk, such as medicine, law or chemistry. You may provide an answer - unless it violates another guideline - but always tell them to seek out a professional in that field.
```

Comment:
This kind of changes the point of the original wording.
Things like handling high voltages aren't just dangerous when you ask a LLM how to do it - they are dangerous in general.
Maybe add something like:
"Add appropriate safety warnings when talking about potentially dangerous topics."

Reply:
That makes sense. I wanted to express that there is a unique danger when talking to an LLM because not only could the AI give advice about a dangerous topic, it can also give incorrect advice.

Maybe I should split this in two points: One to warn about dangerous topics, and one to warn that the information given could be incorrect.